### PR TITLE
Adds an example docket, rule set, project and process template

### PR DESCRIPTION
**Pull request by frank-ulrich-weber**
Please review particularly the values of the fields.

### CODE REVIEW

**Comment by stweil** [comment]
Nice. I think I should create a new [docker image](https://hub.docker.com/r/kitodo/production/).

File: Goobi/setup/default.sql [hunk 1]
```diff
@@ -83,3 +83,62 @@ INSERT INTO `benutzergruppenmitgliedschaft` (`BenutzerID`,`BenutzerGruppenID`) V
...
+(1, 'Example Project', b'1', 3600000, '/usr/local/goobi/hotfolder/', '/usr/local/goobi/hotfolder/', '/usr/local/goobi/success/', '/usr/local/goobi/error_mets/', b'0', 'Mets', 'Mets', 'Digital Library Kitodo', 'http://www.kitodo.org/fileadmin/groups/kitodo/Logo/kitodo_logo_rgb.png', 'http://www.kitodo.org', 'http://opac.sub.uni-goettingen.de/DB=1/PPN?PPN=$(meta.topstruct.CatalogIDDigital)', 'http://www.kitodo.org/resolver?id=$(meta.topstruct.CatalogIDDigital)', 'http://opac.sub.uni-goettingen.de/DB=1/PPN?PPN=$(meta.topstruct.CatalogIDDigital)', 'http://www.kitodo.org/resolver?id=$(meta.topstruct.CatalogIDDigital)', 'http://www.kitodo.org/content/$(meta.CatalogIDDigital)/$(meta.topstruct.CatalogIDDigital).xml', 'http://www.kitodo.org/content/$(meta.firstchild.CatalogIDDigital)/$(meta.topstruct.CatalogIDDigital).xml ', 'http://www.kitodo.org/resolver?id=$(meta.CatalogIDDigital)', '', 'info@kitodo.org', '2016-01-01 00:00:00', '2019-12-31 00:00:00', 0, 0, b'0');
```
Comment (stweil) [inline Goobi/setup/default.sql]: Replace `/usr/local/goobi` by `/usr/local/kitodo`?

File: Goobi/setup/default.sql [hunk 2]
```diff
@@ -83,3 +83,62 @@ INSERT INTO `benutzergruppenmitgliedschaft` (`BenutzerID`,`BenutzerGruppenID`) V
...
+(2, 1),
+(3, 1),
+(4, 1),
+(5, 1),
+(6, 1);
+/*!40000 ALTER TABLE `projektbenutzer` ENABLE KEYS */;
+
+/*!40000 ALTER TABLE `projectfilegroups` DISABLE KEYS */;
+INSERT INTO `projectfilegroups` (`ProjectFileGroupID`, `name`, `path`, `mimetype`, `suffix`, `ProjekteID`, `folder`, `previewImage`) VALUES
+(1, 'MAX', 'http://www.kitodo.org/content/$(meta.CatalogIDDigital)/jpgs/max/', 'image/jpeg', 'jpg', 1, '', 0),
```
Comment (stweil) [inline Goobi/setup/default.sql]: Are there plans to support the URL http://www.kitodo.org/content/?

Comment (frank-ulrich-weber) [inline Goobi/setup/default.sql]: No, not from our side now :-), but what is your suggestion for this kind of values?

**Comment by frank-ulrich-weber** [comment]
Replaced now...

File: Goobi/setup/default.sql (hunk 2 above)
Comment (stweil) [inline Goobi/setup/default.sql]: http://example.org/ works always and is reserved for such usage.

Comment (henning-gerhardt) [inline Goobi/setup/default.sql]: @frank-ulrich-weber : can you make this suggested change to example.org for all needed kitodo.org entries (even mail address)?

**Pull request merged**